src/gigs-board/pages/Create.jsx

```diff
@@ -471,7 +471,7 @@ function generateDescription(text, amount, token, supervisor) {
 
 return (
   <div class="bg-light d-flex flex-column flex-grow-1">
-    {widget("components.layout.app-header")}
+    {widget("components.organism.app-header")}
```

Comment:
I just realized of one potential fail path with moving files: Suppose one file is moved but not all paths are updated, it can work ok because of historical deployment. But when deploy to new it can fail. So double check all renamed widget reference are updated, and it would be best to deploy everything to a new account before merge.